napari/_qt/dialogs/qt_plugin_dialog.py

```diff
@@ -623,7 +881,9 @@ def _add_to_installed(distname, enabled, npe_version=1):
             self._warn_dialog.exec_()
 
     def setup_ui(self):
-        self.resize(1080, 640)
+        """Defines the layout for the PluginDialog."""
+
+        self.resize(950, 640)
```

Comment:
Can w store this information in napari settings? To allow users to adjust it to their preference.

Reply:
yes, can I add this one to the "to do" list?

Reply:
ok